exporter/opencensusexporter/factory_test.go

```diff
@@ -15,17 +15,23 @@
 package opencensusexporter
 
 import (
+	"context"
+	"github.com/open-telemetry/opentelemetry-service/exporter/exportertest"
```

Comment:
Please fix ordering of imports as per our usual convention.

Reply:
thanks, I need to proper setup goland to avoid those...

Reply:
If you manage to convince goland let me know how. I wasn't able to, it messes up the order constantly.